backend/Ript-Fitness-Backend/src/main/java/com/riptFitness/Ript_Fitness_Backend/domain/model/ExerciseModel.java

```diff
@@ -0,0 +1,111 @@
+package com.riptFitness.Ript_Fitness_Backend.domain.model;
+
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;  // Correct import for JPA
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+
+
+// TODO: Add the following fields to the class:
+// - ExerciseID (auto incerment) *
+// - sets *
+// - reps *
+// - weight *
+// - nameOfExercise *
+// - reference_user_id *
+// - refernce_workout_id (set after the workout has been set) * 
+@Entity
+@Table(name = "exercise_model")
+public class ExerciseModel {
```

Comment:
Please call future data models as just "Exercise" instead of including the word "model", it goes more with the rules of the architecture than anything actually wrong. You also won't need the "@Table" annotation in that case.